### PR TITLE
sql_homeworkWeek1.sql

### PR DESCRIPTION
**Pull request by SlaviaF**

### CODE REVIEW

File: databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+-- Get the single most recently created task
+SELECT *
+FROM task
+ORDER BY created DESC
+Limit 0, 1;
+
+-- Get the title and due date of all tasks where the title or description contains database
+SELECT title, due_date
+FROM task
+WHERE description LIKE "%database%" OR title LIKE "%database%";
```
Comment (sboutzen) [inline databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql]: In mysql the `LIKE` is not case insensitive. Unless you are positive you only want this word with this exact case, you could use `WHERE LOWER(description) LIKE "%database%" OR LOWER(title) LIKE "%database%";`

File: databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+WHERE status.name="done";
+
+-- Find all the tasks that are not marked as done
+SELECT task.id, task.title, task.description, status.name
+FROM task
+INNER JOIN status ON status.id=task.status_id
+WHERE status.name!="done";
+
+-- Get all the tasks, sorted with the most recently created first
+SELECT *
```
Comment (sboutzen) [inline databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql]: Avoid * selection. In some cases select * can impact how rows are locked. It is better to explicitly write the column names you want

File: databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+SELECT COUNT(task.status_id), status.name
+FROM task
+INNER JOIN status ON status.id=task.status_id
+GROUP BY task.status_id;
+
+-- Get the names of all statuses, sorted by the status with most tasks first
+SELECT task.title, task.created, status.name
+FROM task
+INNER JOIN status ON status.id=task.status_id
+ORDER BY status.name,
```
Comment (sboutzen) [inline databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql]: It seems you need to have the count here in order to actually group by the ones having the most.

**Review by sboutzen — COMMENTED** [review]
Well done! Some small comments :)

File: databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql (hunk 1 above)
Comment (SlaviaF) [reply databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql]: I wasn't aware it is case sensitive. Thanks for letting me know

File: databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql (hunk 2 above)
Comment (SlaviaF) [reply databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql]: will make the changes

File: databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql (hunk 3 above)
Comment (SlaviaF) [reply databases/week1/SQL_class_HomeWORK/sql_homeworkWeek1.sql]: Ohhh I didnt understand the question right . Have made the changes

**Pull request merged**